src/main/java/seedu/bankwithus/exceptions/CommandNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package seedu.bankwithus.exceptions;
+
+/**
+ * Thrown when command is not recognised
+ */
+public class CommandNotFoundException extends Exception {
+
+}
```

Comment:
Refer to issue https://github.com/AY2223S2-CS2113-T13-3/tp/issues/32 for details